lsp-haskell.el

```diff
@@ -74,16 +74,88 @@
   :group 'lsp-haskell
   :type 'boolean)
 (defcustom lsp-haskell-formatting-provider
-  "ormolu" 
+  "ormolu"
   "The formatter to use when formatting a document or range."
   :group 'lsp-haskell
-  :type '(choice (const :tag "brittany" "brittany") 
-                 (const :tag "floskell" "floskell") 
-                 (const :tag "fourmolu" "fourmolu") 
-                 (const :tag "ormolu" "ormolu") 
-                 (const :tag "stylish-haskell" "stylish-haskell") 
+  :type '(choice (const :tag "brittany" "brittany")
+                 (const :tag "floskell" "floskell")
+                 (const :tag "fourmolu" "fourmolu")
+                 (const :tag "ormolu" "ormolu")
+                 (const :tag "stylish-haskell" "stylish-haskell")
                  (const :tag "none" "none")))
 
+;; ---------------------------------------------------------------------
+;; Plugin-specific configuration
+(defgroup lsp-haskell-plugins nil
+  "Customization group for 'lsp-haskell' plugins."
```

Comment:
```suggestion
  "Customization group for 'haskell-language-server' plugins."
```

They're not `lsp-haskell` plugins, after all!

Reply:
well, the elisp package is called `lsp-haskell`